apps/marketplace/main.scss

```diff
@@ -14,6 +18,22 @@
   font-size: 1.25rem;
 }
 
+.pageAlert {
```

Comment:
These get overwritten by Design System (in current form).  I've added it to this file so that these classes can be applied to other page alerts if required.